kaldb/src/test/java/com/slack/kaldb/testlib/MetricsUtil.java

```diff
@@ -1,17 +1,50 @@
 package com.slack.kaldb.testlib;
 
 import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.search.MeterNotFoundException;
 
 public class MetricsUtil {
+
   public static double getCount(String counterName, MeterRegistry metricsRegistry) {
-    return metricsRegistry.get(counterName).counter().count();
+    try {
+      return metricsRegistry.get(counterName).counter().count();
+    } catch (MeterNotFoundException e) {
```

Comment:
Log the MeterNotFoundException and return 0 for all these functions. 

Also. log other exceptions and re-throw so we can easily see error in log in the future.